qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -92,13 +92,22 @@
                 - none:
                     fs_binary_extra_options += ",cache=none"
         - socket_group:
-            only socket_group.default.default
+            no extra_parameters
             socket_group_test = yes
             socket_group = viofs_g
             fs_binary_extra_options += " --socket-group=viofs_g"
-            pre_command = groupadd -f ${socket_group}
-            post_command = groupdel ${socket_group}
-            cmd_get_sock = "ls -l %s"
+            variants:
+                - @default:
+                    pre_command = groupadd -f ${socket_group}
+                    post_command = groupdel ${socket_group}
+                    cmd_get_sock = "ls -l %s"
+                - unknown_group:
+                    type = virtio_fs_bad_socket_group_name
+                    unknown_socket_group_test = yes
```

Comment:
Sorry, where is this identifier called? I didn't find it.

Reply:
It's useless, thank you.